src/appdistribution/options-parser-util.ts

```diff
@@ -62,3 +64,105 @@ export function getAppName(options: any): string {
   const appId = options.app;
   return `projects/${appId.split(":")[1]}/apps/${appId}`;
 }
+
+/**
+ * Takes in comma separated string or a path to a comma/new line separated file
+ * and converts the input into a string[] of test device strings. Value takes precedent
+ * over file.
+ */
+export function getTestDevices(value: string, file: string): TestDevice[] {
+  // If there is no value then the file gets parsed into a string to be split
+  if (!value && file) {
+    ensureFileExists(file);
+    value = fs.readFileSync(file, "utf8");
+  }
+
+  if (!value) {
+    return [];
+  }
+
+  // The value is split into a string[]
```

Comment:
Nit: This comment isn't very helpful - it basically just explains the type signature of `.split()`, which well known, but doesn't explain what the rest of this  chain does. Consider removing it or improving it.

Reply:
Removed